src/navigator/DrawerNavigator.tsx

```diff
@@ -261,7 +261,7 @@ export default function DrawerNavigator() {
         celoMenuItem
       )}
 
-      {dappsListUrl && (
+      {!!dappsListUrl && (
```

Comment:
This bug is easy to overlook. When the string is empty, it returns it. But in RN text is only valid within `<Text>` components.

It's likely a linting rule could help us catch this too.